app/assets/javascripts/cascade/level/rotator.js

```diff
@@ -31,7 +31,7 @@ $(function () {
         var beforeSlideChange = function(slider) {
             if ( !isSliderMasthead ) { return; }
 
-            $header.fadeOut();
```

Comment:
I wasn't sure what this difference was. [This StackOverflow comment](http://stackoverflow.com/questions/3311758/fadein-vs-fadeout-vs-fadeto#comment41672608_3311790) explains it: `fadeTo` doesn't tweak the `display` property.